tests/data_files/Readme-x509.txt

```diff
@@ -91,3 +95,13 @@ Signing CA in parentheses (same meaning as certificates).
 
 Note: crl_future would revoke server9 and cert_sha384.crt if signed by CA 1
       crl-rsa-pss* would revoke server6.crt if signed by CA 2
+
+Generation
+----------
+
+Newer test files have been generated through commands in the Makefile. The
+resulting files are committed to the repository so that the tests can
+run without having to re-do the generation and so that the output is the
+same for everyone (the generation process is randomized).
```

Comment:
This looks a little complicated. I mean, is there a strong reason why we would like to add the maintenance burden on another Makefile so that we can randomly generate the same test? Why cant we just keep the files you generated and record their origin?